djangocms_admin_style/sass/partials/_tables.scss

```diff
@@ -19,8 +19,37 @@ table {
     tfoot td {
         background: $background-color-ultralight;
     }
-    thead th.sortable:hover {
-        background: $background-color-light;
+    thead th.sorted .sortoptions a {
+        &.ascending {
+            background: url('../img/sorting-icons.png') -5px -50px no-repeat;
+            &:hover {
+                background: url('../img/sorting-icons.png') -5px -72px no-repeat;
+            }
+        }
+        &.descending {
+            background: url('../img/sorting-icons.png') -5px -94px no-repeat;
+            &:hover {
+                background: url('../img/sorting-icons.png') -5px -115px no-repeat;
+            }
+        }
+        &.sortremove {
+            background: url('../img/sorting-icons.png') -4px -5px no-repeat;
+            &:hover {
+                background: url('../img/sorting-icons.png') -4px -26px no-repeat;
```

Comment:
We use double quotes: https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/.csscomb.json#L9